modules/installation-configuration-parameters.adoc

```diff
@@ -1096,13 +1105,33 @@ Additional GCP configuration parameters are described in the following table:
 |Any valid region name, such as `us-central1`.
 
 |`platform.gcp.controlPlaneSubnet`
-|The name of the existing subnet in your VPC that you want to deploy your control plane machines to.
+|The name of the existing subnet where you want to deploy your control plane machine.
 |The subnet name.
 
 |`platform.gcp.computeSubnet`
-|The name of the existing subnet in your VPC that you want to deploy your compute machines to.
+|The name of the existing subnet where you want to deploy your compute machines.
 |The subnet name.
 
+|`platform.gcp.createFirewallRules`
+|Optional. Set this value to `Disabled` if you want to create and manage your firewall rules. By default, the cluster will automatically create and manage the firewall rules that are required for cluster communication. Your service account must have `roles/compute.networkAdmin` and `roles/compute.securityAdmin` privileges in the host project to perform these tasks automatically. 
+|`Enabled` or `Disabled`. The default value is `Enabled`.
+
+|`platform.gcp.publicDNSZone.id`
+|Optional. The name of an existing public DNS zone in the host project. If you set this value, your service account must have the `roles/dns.admin` privilege in the host project. The public DNS zone domain must match the `baseDomain` parameter. If you do not set this value, the installation program will use the public DNS zone in the service project.
```

Comment:
Suggestion: `Optional. ID or name of the public DNS zone. `, and tell other info with `platform.gcp.publicDNSZone.project`.

Reply:
I reworded the public and private DNS zone parameters to take this feedback into account. What do you think?

Reply:
> I reworded the public and private DNS zone parameters to take this feedback into account. What do you think?

@jianli-wei Any additional feedback before we move forward with this PR? Thank you